CoinMarketCap/Client/CryptocurrencyClient.cs

```diff
@@ -334,7 +334,129 @@ public class CryptocurrencyClient : ApiClientBase
                     ["aux"] = aux
                 });
 
-        }
-        #endregion Endpoint: /v1/cryptopcurrency/quotes/historical
+        }
+        #endregion Endpoint: /v1/cryptopcurrency/quotes/historical - Historical Quotes
+
+        #region Endpoint: /v1/cryptocurrency/market-pairs/latest - Market Pairs Latest
+
+        /// <summary>
+        /// <see cref="MarketPairsLatest"/>
+        /// </summary>
+        /// <param name="id">A cryptocurrency or fiat currency by CoinMarketCap ID to list market pairs for. Example: "1"</param>
+        /// <param name="start">Optionally offset the start (1-based index) of the paginated list of items to return. Default is 1.</param>
+        /// <param name="limit">Optionally specify the number of results to return. Use this parameter and the "start" parameter to determine your own pagination size.</param>
+        /// <param name="sortDir">Optionally specify the sort direction of markets returned. Default is "desc".</param>
+        /// <param name="sort">Optionally specify the sort order of markets returned. By default we return a strict sort on 24 hour reported volume. Pass cmc_rank to return a CMC methodology based sort where markets with excluded volumes are returned last.</param>
+        /// <param name="aux">Optionally specify a comma-separated list of supplemental data fields to return. Pass num_market_pairs,category,fee_type,market_url,currency_name,currency_slug,price_quote,notice to include all auxiliary fields.</param>
+        /// <param name="matchedId">Optionally include one or more fiat or cryptocurrency IDs to filter market pairs by. For example ?id=1&matched_id=2781 would only return BTC markets that matched: "BTC/USD" or "USD/BTC". This parameter cannot be used when matched_symbol is used.</param>

```

Comment:
Note that these comments are xml, so the ampersands need to be encoded as `&amp;` - please review all of the xmldoc comments and be sure to encode any xml entities. When viewing the diff on this page, they appear highlighted red.